app/config/routes.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  vetVisits: process.env.DASHBOARD_SERVICE_URI,
```

Comment:
Better to take this from config, the pattern we follow is to load all the environment variables into the config - that way we also set defaults and restrictions

Reply:
Thanks Alex, I've updated.